src/PlanetWarsProxy.py

```diff
@@ -151,115 +191,86 @@ def EnemyFleets(self):
             if fleet.Owner() != self.PlayerID():
                 r.append(fleet)
         return r
-
-    def _ToString(self):
-        s = ''
-        s+= "M %d %d %d %d\n" % \
-            (self._gameid, self._playerid, self._tick, self._winnerid)
-        for p in self._planets:
-            s += "P %f %f %d %d %d\n" % \
-             (p.X(), p.Y(), p.Owner(), p.NumShips(), p.GrowthRate())
-        for f in self._fleets:
-            s += "F %d %d %d %d %d %d\n" % \
-             (f.Owner(), f.NumShips(), f.SourcePlanet(), f.DestinationPlanet(), \
-                f.TotalTripLength(), f.TurnsRemaining())
-        return s
-
-    def Distance(self, source, destination):
-        return source.DistanceTo(destination)
+    #  == Save Function ==
```

Comment:
Nit: generally, if you don't need a method, I tend to delete it, it's saved in the git history if you ever need it again :)